<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPengumuman</class>
 <widget class="QWidget" name="FormPengumuman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Pengumuman</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>&lt;h2&gt;Form Data Pengumuman&lt;/h2&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupInput">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutInput">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelId">
            <property name="text">
             <string>ID Pengumuman (kosong = auto)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="EditId"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSub">
            <property name="text">
             <string>Sub Jenis</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="EditSub"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelIsi">
            <property name="text">
             <string>Isi Pengumuman</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="EditIsi"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridButtons">
          <item row="0" column="0">
           <widget class="QPushButton" name="BtnSimpan">
            <property name="text">
             <string>SIMPAN</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="BtnUbah">
            <property name="text">
             <string>UBAH</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="BtnHapus">
            <property name="text">
             <string>HAPUS</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="BtnBersih">
            <property name="text">
             <string>BERSIH</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="spacerInput">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupData">
       <property name="title">
        <string>Data</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutData">
        <item>
         <layout class="QHBoxLayout" name="layoutCariCetak">
          <item>
           <widget class="QLabel" name="labelCari">
            <property name="text">
             <string>Cari:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineCari">
            <property name="placeholderText">
             <string>Ketik untuk mencari...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCetak">
            <property name="text">
             <string>CETAK PDF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tblPengumuman">
          <property name="columnCount">
           <number>3</number>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Sub Jenis</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Isi</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
